src/sentry/api/endpoints/issues_resolved_in_release.py

```diff
@@ -61,5 +61,13 @@ def get(self, request, project, version):
 
         groups = Group.objects.filter(project=project, id__in=group_ids)
 
-        context = serialize(list(groups), request.user, StreamGroupSerializer(stats_period=None))
+        context = serialize(
+            list(groups),
+            request.user,
+            StreamGroupSerializer(
+                stats_period=None,
```

Comment:
These aren't really specific to this change, but:

- `stats_period` defaults to `None` and doesn't need to be provided here,
- the behavior of `GroupSerializer` and `StreamGroupSerializer` is only differentiated when `stats_period` or `matching_event_id` is not `None`. Neither of those are the case here, so shouldn't this use `GroupSerializer`?

We should go through and clean these up later.

Reply:
Or here too this works 😎

Reply:
I like to keep my contribution graph nice and low.